packages/logger/src/index.ts

```diff
@@ -35,8 +35,8 @@ export const getLogger = (path: string) =>
           format.printf((msg) =>
             formatMessage({
               level: msg.level,
-              message: msg.message,
-              timestamp: msg.timestamp,
+              message: msg.message as string,
+              timestamp: msg.timestamp as string,
```

Comment:
Maybe not that sound but it's OK for now. The entire logger stuff will have to be looked into more w/ OTEL or other telemetry stuff later